backend/serializers.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import User
+
+from rest_framework import serializers
+from rest_auth.registration.serializers import RegisterSerializer
+
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import setup_user_email
+
+class AuthRegisterSerializer(RegisterSerializer):
+
+    def get_cleaned_data(self):
+        return {
+            'username': self.validated_data.get('username', ''),
+            'password1': self.validated_data.get('password1', ''),
```

Comment:
why is this password**1** ?

Reply:
it is the default way of handling password from the library. It expects "password1" as the first password, and "password2" as "confirmed password". This could get changed if we need to later on.